contracts/eosio.system/src/eosio.system.cpp

```diff
@@ -109,6 +109,15 @@ namespace eosiosystem {
       _gstate2.new_ram_per_block = bytes_per_block;
    }
 
+   void system_contract::channel_to_system_fee( const name& from, const asset& amount, const name& protocol, const std::string& memo ) {
+         system_contract::logsystemfee_action logsystemfee_act{ get_self(), { {get_self(), active_permission} } };
+         logsystemfee_act.send(protocol, amount, memo);
+         // inline transfer to fee_account
+         token::transfer_action transfer_act{ token_account, { from, active_permission } };
+         transfer_act.send( from, fee_account, amount,
```

Comment:
Perhaps best to move the transfer event prior to the logging

Would make a bit more logical sense to have the logging event after the transfer event.